api.go

```diff
@@ -2923,7 +2954,38 @@ func (sc *Client) CreateOrUpdateEdgeDeploymentService(corpName, siteName, fastly
 		return err
 	}
 
-	_, err = sc.doRequest("PUT", fmt.Sprintf("/v0/corps/%s/sites/%s/edgeDeployment/%s", corpName, siteName, fastlySID), string(b))
+	var sleep_time = 4 * time.Second
```

Comment:
To match the standard [Go style](https://google.github.io/styleguide/go/guide#mixedcaps):
```suggestion
	var sleepTime = 4 * time.Second
```